radixdlt-core/radixdlt/src/main/java/com/radixdlt/statecomputer/checkpoint/GenesisProvider.java

```diff
@@ -121,6 +122,7 @@ public VerifiedTxnsAndProof get() {
 			// Initial validator registration
 			for (var validatorKey : validatorKeys) {
 				genesisTxnConstruction.registerAsValidator(validatorKey.getPublicKey());
+				genesisTxnConstruction.action(new UpdateAllowDelegationFlag(validatorKey.getPublicKey(), true));
```

Comment:
I thought there was a transition procedure which allows delegation if `AllowDelegationFlag` is missing, no?